src/Persistent/Api.hs

```diff
@@ -148,16 +149,15 @@ newtype PersistExcept = PersistExcept { unPersistExcept :: ServantErr }
 class (MonadError PersistExcept m, Monad m) => PersistM m where
     getDb :: AulaGetter a -> m a
     modifyDb :: AulaSetter a -> (a -> a) -> m ()
-
     getCurrentTimestamp :: m Timestamp
+    mkRandomPassword :: m UserPass
 
-    default getCurrentTimestamp :: MonadIO m => m Timestamp
```

Comment:
The `DefaultSignatures` language extension is not needed any more. Please remove it.